### PR TITLE
[vs] Make default port admin state to false

### PR DESCRIPTION
**Pull request by kcudnik**

### CODE REVIEW

File: vslib/src/SwitchStateBase.cpp [hunk 1]
```diff
@@ -378,6 +378,13 @@ sai_status_t SwitchStateBase::setPort(
 
             if (kvp.second->m_portId == portId)
             {
+                if (ifup(tapname.c_str(), portId, up, false))
+                {
+                    SWSS_LOG_ERROR("if admin %s failed on %s failed: %s", (up ? "UP" : "DOWN"), tapname.c_str(),  strerror(errno));
+
+                    return SAI_STATUS_FAILURE;
+                }
```
Comment (lguohan) [inline vslib/src/SwitchStateBase.cpp:386]: this 381 - 386 should be removed. not neccessary.

File: vslib/src/SwitchStateBaseHostif.cpp [hunk 2]
```diff
@@ -611,15 +611,34 @@ sai_status_t SwitchStateBase::vs_create_hostif_tap_interface(
...
+
+    if (status != SAI_STATUS_SUCCESS)
+    {
+        SWSS_LOG_ERROR("failed to get admin state for port %s",
+                sai_serialize_object_id(obj_id).c_str());
+
+        return false;
+    }
+
+    if (ifup(name.c_str(), obj_id, attr.value.booldata, false))
```
Comment (lguohan) [inline vslib/src/SwitchStateBaseHostif.cpp:628]: this should the vname.

**Comment by lguohan** [comment]
this does not work. check hostif_create_tap_veth_forwarding(), you always bring the link up there. Can you fix that function?

**Comment by kcudnik** [comment]
I fixed that, but that was introduced also on your request to have notifications when switch is instantiated

File: vslib/src/SwitchStateBaseHostif.cpp [hunk 3]
```diff
@@ -479,7 +479,7 @@ bool SwitchStateBase::hostif_create_tap_veth_forwarding(
 
     SWSS_LOG_NOTICE("interface index = %d, %s\n", sock_address.sll_ifindex, vethname.c_str());
 
-    if (ifup(vethname.c_str(), port_id, true, true))
+    if (ifup(vethname.c_str(), port_id, false, false))
```
Comment (lguohan) [inline vslib/src/SwitchStateBaseHostif.cpp:482]: why do you need this line at all? you have set the admin state right before enter this function.

**Comment by kcudnik** [comment]
you are right, not needed

**Pull request merged**